### ISSUE TITLE
Add a new star() method to draw regular stars!

### ISSUE THREAD

**Issue by Lucas-C** [PyFPDF/fpdf2#428]
The idea comes from one of @digidigital recipes: https://github.com/digidigital/Extensions-and-Scripts-for-pyFPDF-fpdf2/tree/main/star (_cf._ #274)

There this some usage example once implemented, inspired by [regular_polygon()](https://pyfpdf.github.io/fpdf2/fpdf/fpdf.html#fpdf.fpdf.FPDF.regular_polygon):
```python
from fpdf import FPDF

pdf = FPDF()
pdf.star(x=100, y=100, r=100, corners=5, rotateDegrees=90, style="FD")
```

By implementing this feature you, as a benevolent [FLOSS](https://en.wikipedia.org/wiki/Free/libre_and_open-source_software) developper, will provide access to the large community of fpdf2 users to a standard and useful PDF functionality.
As a contributor you will get review feedbacks from maintainers & other contributors, and learn about the lifecycle & structure of a Python library on the way.
You will also be added into the [contributors list & map](https://github.com/PyFPDF/fpdf2#contributors-).

**Comment by RedShy** [comment]
Hi! I would like to work on this! Is there any feature to add other than the rotation?

**Comment by Lucas-C** [comment]
> I would like to work on this!

Great!

> Is there any feature to add other than the rotation?

Actually, the `rotateDegrees` is really optional:
we already have a [rotation()](https://pyfpdf.github.io/fpdf2/fpdf/fpdf.html#fpdf.fpdf.FPDF.rotation) method so users will be able to:
```python
with pdf.rotation(...):
    pdf.star(...)
```
I suggested to introduce it just because [regular_polygon()](https://pyfpdf.github.io/fpdf2/fpdf/fpdf.html#fpdf.fpdf.FPDF.regular_polygon) already has it.

No, I don't see anything else to add, but feel free to make suggestions! 😊

Also, as with all methods accepting a `style` argument, unit tests should ensure that the line color & fill color can be configured.

**Comment by Lucas-C** [comment]
Implemented in #434
Thank you @RedShy !
